styles/variables.scss

```diff
@@ -150,6 +150,7 @@ $image-placeholder-blur-enabled: false;
 
 // custom colors
 $lightpink: lightpink;
+$white: #ffffff;
```

Comment:
? $white = white no need for this variable

Reply:
yeah but its being used in a lot of places so I thought we can have something common so in future if we need to change this to some other variable, it would be easier? I can remove though if you think it's useless?

Reply:
@prachi00 I think $white already exist on [bulma](https://bulma.io/documentation/customize/variables/#initial-variables), you are overriding initial variable

Reply:
> @prachi00 I think $white already exist on [bulma](https://bulma.io/documentation/customize/variables/#initial-variables), you are overriding initial variable

oh it does, my bad